src/DataProtection/DataProtection/src/AuthenticatedEncryption/AuthenticatedEncryptorFactory.cs

```diff
@@ -54,7 +54,7 @@ public IAuthenticatedEncryptor CreateEncryptorInstance(IKey key)
                     throw new PlatformNotSupportedException(Resources.Platform_WindowsRequiredForGcm);
                 }
 
-                Debug.Assert(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
+                Debug.Assert(OperatingSystem.IsWindows());
```

Comment:
DataProtection still cross-compiles to netstandard2.0 which does not have this API. Could you undo changes to files under that directory?